src/index.js

```diff
@@ -4,9 +4,20 @@ const path = require('path');
 const util = require('util');
 const TapMochaReporter = require('tap-mocha-reporter');
 
-async function findBrsFiles(sourceDir) {
-    let searchDir = sourceDir || 'source';
-    const pattern = path.join(searchDir, '**', '*.brs');
+async function findBrsFiles(exclusions) {
+    const defaultDirs = ['source', 'components'];
+    let exclusionList = [];
+
+    if (exclusions) {
+        exclusionList = exclusions.trim().split(',');
+    }
```

Comment:
Neato!  Is this so users can exclude their `tests/` directory (and probably something like `node_modules/`)?  If so, it might be useful to have those as defaults! 😃

Reply:
yep, that's the idea. I'll add those as defaults.